charts/karmada/README.md

```diff
@@ -121,7 +121,15 @@ Execute command (switch to the `root` directory of the repo, and sets the `curre
 kubectl config use-context member
 helm install karmada-agent -n karmada-system --create-namespace ./charts/karmada
 ```
+
 ### 2. Install component
+Before installing estimator, we should set flags of scheduler to take effect:
```

Comment:
Hi @calvin0327, I guess you misunderstood what I mean. We do not need to add specific words for a paramter, just add it in https://github.com/karmada-io/karmada/tree/master/charts/karmada#common-parameters is ok.

Reply:
Sorry. updated